README.md

```diff
@@ -151,7 +151,34 @@ By default, the Kogito services are built with traditional `java` compilers to s
 
 Kogito services implemented with [Quarkus](https://quarkus.io/guides/kogito-guide) can be built to native binary. This means very low footprint on runtime (see [performance examples](https://www.graalvm.org/docs/examples/java-performance-examples/)), but a lot of resources during build time. For more information about AOT compilation, see [GraalVM Native Image](https://www.graalvm.org/docs/reference-manual/aot-compilation/).
 
-In Kogito Operator tests, native builds take approximately 10 minutes and the build pod can consume up to 10GB of RAM and 1.5 CPU cores. Ensure that you have these resources available when running native builds.
+In Kogito Operator tests, native builds take approximately 10 minutes and the build pod can consume up to 10GB of RAM and 1.5 CPU cores.
+
+:warning: By default Kogito application doesn't contain resource requests or limits. This may lead to the situation when native build is terminated due to insufficient memory. To prevent this behaviour user can specify at least memory request for Kogito application build, making sure that build pod is allocated on OpenShift node with enough of free memory. The side effect of this configuration is that OpenShift will prioritize the build pod.
```

Comment:
Please add the link to the OKD documentation that explains the pod prioritization when resources limits/requests are set. Thanks!!

Reply:
Done.